korman/operators/op_mesh.py

```diff
@@ -387,6 +387,120 @@ def origin_to_bottom(obj):
     mw.translation = global_origin
 
 
+class PlasmaAddLinkingBookMeshOperator(PlasmaMeshOperator, bpy.types.Operator):
+    bl_idname = "mesh.plasma_linkingbook_add"
+    bl_label = "Add Linking Book"
+    bl_category = "Plasma"
+    bl_description = "Adds a new Plasma Linking Book"
+    bl_options = {"REGISTER", "UNDO"}
+
+    anim_offsets = {
+        "Stand": (0.2, 1.7, -3.4),
+        "Kneel": (0.8, 2.5, -1.8),
+    }
+
+    # Allows user to specify their own name stem
+    panel_name = bpy.props.StringProperty(name="Name",
+                                         description="Linking Book name stem",
+                                         default="LinkingBook",
+                                         options=set())
+    link_anim_type = bpy.props.EnumProperty(name="Link Animation",
+                                            description="Type of Linking Animation to use",
+                                            items=[("Stand", "Standing", "The avatar steps up to the book and places their hand on the panel"),
+                                                   ("Kneel", "Kneeling", "The avatar kneels in front of the book and places their hand on the panel"),],
```

Comment:
Maybe it would be better to use the animation names: `LinkOut`, `FishBookLinkOut` as the keys.